themes/vue/layout/partials/support_vue_dropdown.ejs

```diff
@@ -1,10 +1,10 @@
 <li class="nav-dropdown-container support-vue">
-  <a class="nav-link">Support Vue</a><span class="arrow"></span>
+  <a class="nav-link">Hỗ trợ Vue</a><span class="arrow"></span>
   <ul class="nav-dropdown">
     <li><ul>
-      <li><a href="https://vue.threadless.com" target="_blank" class="nav-link">Shop</a></li>
-      <li><a href="https://opencollective.com/vuejs" target="_blank" class="nav-link">Support Us on OpenCollective</a></li>
-      <li><a href="https://www.patreon.com/evanyou" target="_blank" class="nav-link">Support Evan on Patreon</a></li>
+      <li><a href="https://vue.threadless.com" target="_blank" class="nav-link">Cửa hàng</a></li>
```

Comment:
"Cửa hàng" sounds unnatural to us Vietnamese. How about "Tặng phẩm" or "Lưu niệm" etc. instead?

Reply:
I think "Tặng phẩm" would be more appropriate to talk about gifts, the same with "Lưu niệm", I'm still looking for another suitable word. The team, please give suggestions?

Reply:
Thật ra thì mình không "tặng," nên thật ra anh cũng không biết. Chắc "cửa hàng" vậy.